client/components/course/Settings/index.vue

```diff
@@ -49,26 +58,32 @@
 
 <script>
 import api from '../../../api/course';
+import CircularProgress from 'components/common/CircularProgress';
 import CloneModal from './CloneModal';
 import EventBus from 'EventBus';
 import General from './General';
 import JSZip from 'jszip';
 import { mapActions, mapGetters } from 'vuex-module';
 import saveAs from 'save-as';
+import confirmPublishing from 'components/common/mixins/publish';
```

Comment:
This is  in general publishing mixin not just confirmation. Please reflect that with naming.